web/pages/cart/cart.jsx

```diff
@@ -0,0 +1,73 @@
+import { useContext } from 'react'
+import './cart.css'
+import { CartContext } from '../../context/CartContext.jsx'
+
+export function Cart() {
+  const { cart, cartCount } = useContext(CartContext)
+
+  console.log(cart)
+  return (
+    <>
+      <div className="container title-container py-3">
+        <h1>Your Cart</h1>
+      </div>
+      <div className="container">
+        <div className="row">
+          <div className="col-12 col-md-8">
+            <ul className="list-group cart-items">
+              {cart.map((item) => (
+                <li key={item.id} className="list-group-item">
+                  {/* Render each cart item here */}
```

Comment:
The cart cards are not rendered properly. @georgevarghese185 I will make changes this PR itself.